From 6f6070dded7d66a410b813ceac03c7b0a451675b Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Thu, 12 Feb 2026 16:32:08 +0100
Subject: [PATCH] python-uci: add python setuptools dependency

Added dependency on python-setuptools, fixing error:
BackendUnavailable: Cannot import 'setuptools.build_meta'
---
 lang/python/python-uci/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/lang/python/python-uci/Makefile b/lang/python/python-uci/Makefile
index 0a539e323..5e577f9aa 100644
--- a/lang/python/python-uci/Makefile
+++ b/lang/python/python-uci/Makefile
@@ -18,9 +18,13 @@ PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
 PKG_LICENSE:=GPL-3.0-or-later
 PKG_LICENSE_FILES:=LICENSE
 
+PKG_BUILD_DEPENDS:=python-setuptools/host python-build/host
+HOST_BUILD_DEPENDS:=python3/host
+
 include ../pypi.mk
 include $(INCLUDE_DIR)/package.mk
 include ../python3-package.mk
+include $(INCLUDE_DIR)/host-build.mk
 
 define Package/python3-uci
   SECTION:=lang
-- 
2.53.0

